fs/aio/aio_write.c

```diff
@@ -277,7 +277,7 @@ int aio_write(FAR struct aiocb *aiocbp)
    */
 
   int flags = fcntl(aiocbp->aio_fildes, F_GETFL);
```

Comment:
flags need move to the begin of function to conform to c89 standard

Reply:
```
int flags;

...

flags = ...;

...